desktop/app/splash.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+// Checks to see if we've launched before (we touch started.txt). If not, write the file and open a browser
+import {socketRoot} from '../../react-native/react/constants/platform.native.desktop'
+import path from 'path'
+
+const helpURL = 'https://keybase.io/docs/cli_kbstage'
+const filePath = path.join(socketRoot, 'started.txt')
+import fs from 'fs'
+import {shell} from 'electron'
+
+export default () => {
+  fs.stat(filePath, (err, stats) => {
+    if (err) {
+      try {
+        fs.writeFileSync(filePath, 'hello world')
+      }
+      catch (e) {
+        console.error(`Couldn't mark splash screen shown: ${e}`)
```

Comment:
if we can't write it, let's not show the splash else we could easily get into a state where we show over and over which would be lame